examples/source/e-commerce/Hotel/api.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+const {setMaxAge} = require('@lib/utils/cacheHelpers');
+const utils = require('@lib/utils');
+const rooms = require(utils.project.absolute('/examples/static/samples/json/rooms.json'));
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+
+examples.all('/rooms', (request, response) => {
+  setMaxAge(response, 0);
```

Comment:
max-age can be long as results will always be the same for the given query parameters

Reply:
Do you have any particular duration in mind? A year?

Reply:
make it 1d in case we want to change the sample

Reply:
Alright, PTAL